### ISSUE TITLE
need ability to add ARIA tags

### ISSUE THREAD

**Issue by apcimino** [pojome/elementor#3988]
In order to make sites for accessible users I need the ability to add ARIA tags. Please add this to the roadmap.

**Comment by bainternet** [comment]
@ramiy

**Comment by bainternet** [comment]
@apcimino 

where are you missing it? a specific widget?

**Comment by apcimino** [comment]
Using the pre-built blocks and templates.
I asked support and they told me it was not possible to add. Instead I should submit the request here.

**Comment by apcimino** [comment]
Specifically it would be helpful to add ARIA tags to section tags. For example, adding role=main.

**Comment by ramiy** [comment]
@apcimino this is a very nice idea.

Currently it's not possible out-of-the-box. But we are planing to add an attribute manager. It will allow you to add custom attributes to any element.

**Comment by ramiy** [comment]
@apcimino see #4030

**Comment by arielk** [comment]
Duplicate of #290

**Comment by tannerchung** [comment]
Following! The attribute manager would be really great. I need this specifically for Pinterest data https://css-tricks.com/using-pinterest-data-attributes-and-meta-tags/, will create a new issue for this but I think it falls in the same category.
